recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -34,6 +40,8 @@ add_to_local_conf() {
     base_update_conf_value ${local_conf} SERIAL_CONSOLE "115200 hvc0"
 
     base_update_conf_value ${local_conf} PREFERRED_VERSION_xen "4.9.0+git\%"
+
+    # set folders where rootfs artifacts are stored
```

Comment:
Hmm... This comment is for what exactly?

Reply:
done